src/QsCompiler/Compiler/CompilationLoader.cs

```diff
@@ -719,6 +624,90 @@ public CompilationLoader(SourceLoader loadSources, IEnumerable<string> reference
         {
         }
 
+        private IEnumerable<(LoadedStep, Action<Status>)> InternalRewriteSteps(Uri executingAssembly)
+        {
+            // TODO: The dependencies for rewrite steps should be declared as part of IRewriteStep interface, and we

```

Comment:
In order to not lose track of this, it would be great to create an issue and reference it in the comment.

Reply:
The todo was already there before, I just moved it to a new method.